OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -63,6 +63,10 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Arc in WAngles, two values indicate variable arc.")]
 		public readonly WAngle[] LaunchAngle = { WAngle.Zero };
 
+		[Desc("Altitude where this bullet should explode when reached.",
+			"Negative values allow this bullet to pass cliffs and terrain bumps.")]
+		public readonly WDist ExplodeUnderThisAltitude = WDist.Zero;
```

Comment:
Stopping the projectile from exploding won't affect the other major problem with incorrect firing offsets: the projectile (artwork) will disappear under the ground on one side of the bump, and burst out on the other side.